build.gradle

```diff
@@ -196,6 +196,33 @@ task apiTest(type: Test) {
   setClasspath(sourceSets.integrationTest.runtimeClasspath)
 }
 
+task migrateClaims() {
+  description 'Migrating claims from claim-store database to CCD'
+
+  doLast {
+    print 'I am going to start migration app'
+
+    javaexec {
+      main="-jar";
+      args = [
+        "ccd-claim-migration/build/libs/ccd-claim-migration-1.1.0.jar",
```

Comment:
maybe set `archiveName` so the jar version isn't in there? https://github.com/hmcts/cmc-claim-store/blob/cnp/build.gradle#L266